app/helpers/miq_event_definition_helper.rb

```diff
@@ -18,7 +18,7 @@ def miq_summary_event_definition_policies(record_policies)
         rows.push({
                     :cells   => [{:icon => ed.decorate.fonticon, :value => ed.description}],
                     :title   => _("View this %{model} Policy") % {:model => ui_lookup(:model => ed.towhat)},
-                    :onclick => "DoNav('/miq_policy/show/#{ed.id}');",
+                    :onclick =>  {:url => "/miq_policy/show/#{ed.id}"},
```

Comment:
Super-mega-minor whitespace issue here.
```suggestion
                    :onclick => {:url => "/miq_policy/show/#{ed.id}"},
```